doc/explanation/grouping_options.ipynb

```diff
@@ -0,0 +1,386 @@
+    "adelie_element"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3fe7048d",
+   "metadata": {},
+   "source": [
+    "## `color`/`c`: Vectorized Coloring\n",
+    "\n",
+    "The `color` parameter creates **vectorized coloring** within a single plot element, where each data point is colored based on the category value.\n",
+    "\n",
+    "### What it creates:\n",
+    "- **Single plot element**: One unified [Scatter](inv:holoviews#holoviews.Scatter), [Curve](inv:holoviews#holoviews.Curve), etc.\n",
+    "- **Vectorized coloring**: Each point colored by category\n",
+    "- **Cannot be indexed**: Single element, not separable by category\n",
+    "\n",
+    "### When to use:\n",
+    "- When you want the best performance for large datasets\n",
+    "- When you need smooth, continuous color mapping\n",
+    "- When you don't need to isolate specific categories"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f54ccbed",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plot_color = penguins.hvplot.scatter(\n",
+    "    x='bill_length_mm',\n",
+    "    y='bill_depth_mm',\n",
+    "    color='species',\n",
+    "    title=\"Color: Vectorized Coloring\"\n",
+    ")\n",
+    "plot_color"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d4fb28a9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Alternative syntax using 'c'\n",
+    "plot_c = penguins.hvplot.scatter(\n",
+    "    x='bill_length_mm',\n",
+    "    y='bill_depth_mm',\n",
+    "    c='species',\n",
+    "    title=\"Using 'c' parameter (same result)\"\n",
+    ")\n",
+    "plot_c"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "60b4c757",
+   "metadata": {},
+   "source": [
+    "### Indexing and access:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ab54c88e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Cannot index by category - this won't work:\n",
+    "print(f\"Type of 'color' plot: {type(plot_color)}\")\n",
+    "try:\n",
+    "    adelie_color = plot_color['Adelie']  # This will raise an error!\n",
+    "except Exception:\n",
+    "    print(\"Error: You cannot index a single element by category!\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1626f2c0",
+   "metadata": {},
+   "source": [
+    "## Visual Output Comparison\n",
+    "\n",
+    "Let's compare all three approaches side by side to see the differences:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a1545149",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import panel as pn\n",
+    "\n",
+    "width = 300\n",
+    "\n",
+    "groupby_plot = penguins.hvplot.scatter(\n",
+    "    x='bill_length_mm', y='bill_depth_mm',\n",
+    "    groupby='species', title=\"groupby='species'\",\n",
+    "    frame_width=width, widget_location='bottom_right',\n",
```

Comment:
Add `shared_axes=False` to `groupby_plot` as otherwise updating the widget affects the view of the other plots.